README.md

```diff
@@ -352,3 +354,91 @@ if (result.canExec) {
   await owner.sendTransaction({ to: calldata.to, data: calldata.data });
 }
 ```
+
+In certain scenarios during function execution, user intervention might be necessary. For instance, if a user updates task storage during its run, expecting successful Transaction execution, it may not always be guaranteed. Currently, there are no specific tools available to assist users in managing such cases effectively. One proposed solution is the use of callbacks. These callbacks allow users to wait for the Transaction status before updating the task storage. There are two types of callbacks: onFail, triggered when the transaction fails, and onSuccess, triggered when the transaction is successfully mined.
```

Comment:
I'll remove this as the README should really only focus on the current state where callbacks is available.
This would be more interesting to share in the blog post announcement for example

Reply:
that was to be removed 😬